pkg/volume/volume_linux.go

```diff
@@ -37,10 +37,14 @@ const (
 // fsGroup, and sets SetGid so that newly created files are owned by
 // fsGroup. If fsGroup is nil nothing is done.
 func SetVolumeOwnership(mounter Mounter, fsGroup *int64) error {
+	path := mounter.GetPath()
+	klog.Infof("Setting volume ownership for %v to %v", path, fsGroup)
 
 	if fsGroup == nil {
+		klog.Infof("No fsGroup specified! No permissions fixing will occur.")
```

Comment:
IIRC - this function does not get called at all in many cases when `fsGroup` is not present - https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/csi/csi_mounter.go#L317 and hence logging here may only be useful in some cases. 

The other thing is -if we really want to bubble this message up to the user then we should expose these as events rather than logging to logs which user may not have access to.

Reply:
@gnufied (1)  for the CSI stuff,  would you be ok with me making those logs in the csi_mounter to be  regular level (v=2 , right?)  (2) i didn't want to disrupt the storage logging defaults for this patch, unless specifically the storage-sig *wants* me to put events in, in which case im happy to do so.

Reply:
yeah , after looking more at this, the logic for wether ownership *should* be set is important.  im going to update the patch to make that more transparent.

Reply:
(resolving since now there are logs in the CSI mounter - as well as a log statement in the  SetVolumeOwnership function for the unsupported path)... LMK if need to reopen.